README.md

```diff
@@ -684,6 +684,203 @@ var callbacks = {
 BillForward.addPayVisionForm(formSelector, supportedCardBrands, wpwlOptions, callbacks);
 ```
 
+### Capture Bank Account
+
+For the moment the only bank account capture supported is in `stripe` using `ACH`.
```

Comment:
This styling of `stripe` _may_ be fine; you are referring to an enum constant.
Although, in that situation: the enum to which you refer is unclear.
If some action is required by the user to "use `stripe`", then that deserves an explanation.